docs/Troubleshooting.md

```diff
@@ -0,0 +1,34 @@
+# Unsigned commits in latest are preventing me from pushing my branch
+
+## Documentation index
+
+Please familiarise yourself with our:
+
+- [Code of conduct](https://github.com/bbc/simorgh/blob/latest/.github/CODE_OF_CONDUCT.md)
+- [Code Standards](https://github.com/bbc/simorgh/blob/latest/docs/Code-Standards.md)
+- [Contributing guidelines](https://github.com/bbc/simorgh/blob/latest/CONTRIBUTING.md)
+- [Guide to Code Reviews](https://github.com/bbc/simorgh/blob/latest/docs/Code-Reviews.md)
+- [Github Project Board Guide](https://github.com/bbc/simorgh/blob/latest/docs/Project-Board-Guide.md)
+- [GPG Signing Guide](docs/GPG-Signing-Guide.md)
+- [Primary README](https://github.com/bbc/simorgh/blob/latest/README.md)
+- [Recommended Tools](https://github.com/bbc/simorgh/blob/latest/docs/Recommended-Tools.md)
+- [Troubleshooting](https://github.com/bbc/simorgh/blob/latest/docs/Troubleshooting.md) (you are here)
+
+If for some reason unsigned commits have been merged into the latest branch, possibly because of an admin merge, then you will need to sign any previously unsigned commits.
+
+## How to Sign Previous Commits
+
+The following instructions was taken from [here](https://hyperledger-indy.readthedocs.io/projects/sdk/en/latest/docs/contributors/signing-commits.html).
```

Comment:
Should be 'The following instructions _were_ taken from [here] :)

Reply:
```suggestion
The following instructions were taken from [here](https://hyperledger-indy.readthedocs.io/projects/sdk/en/latest/docs/contributors/signing-commits.html).
```